src/styles/theme.scss

```
@@ -70,3 +82,13 @@ $breakpointTablet: 1024px;
     @content;
   }
 }
+@mixin panelMobileUp {
```

Stylelint (17.13.0)

[error] Expected mixin name "panelMobileUp" to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)